Project 13/index.ts

```diff
@@ -0,0 +1,18 @@
+import {Grid} from "./entity/grid";
+import {Shop} from "./entity/shop";
+import {Page} from "./entity/page";
+
+const shop: Shop = new Shop(
+    <HTMLParagraphElement>document.getElementById('quantity'),
+    <HTMLParagraphElement[]>Array.from(document.getElementsByClassName('amount')),
+    <HTMLDivElement>document.getElementById('cart-preview'),
+    <HTMLDivElement>document.getElementById('shopping-window'),
+    <HTMLTemplateElement>document.getElementById('cart-item-template')
+);
+
+const grid: Grid = new Grid(
```

Comment:
```suggestion
const grid = new Grid(
```
можно не указывать тип: он вычислен тайпскриптом неявно при инициализации филда